jekyll/_cci2/hello-world.md

```diff
@@ -142,7 +142,7 @@ In the top left, you will find the Org switcher.
 {:.tab.switcher.Cloud}
 ![Switch Organization Menu]({{ site.baseurl }}/assets/img/docs/org-centric-ui_newui.png)
 
-{:.tab.switcher.Server}
+{:.tab.switcher.Server v2.19}
```

Comment:
This one is missing the dash.

Should we standardize? You have v2.19, v2.19.x, and v2 - I just don't want users to be confused.

Reply:
Oh weird I thought I had fixed this, maybe I didn't push it up. Let me check! Thanks